File: Proyecto en C#/Entrega Diseno/CategoriaBanda.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Entrega_Diseno
{
    class CategoriaBanda
    {
        private int _Tipo;
        public int Tipo
        {
            get { return _Tipo; }
            set { _Tipo = value; }
        }
        private int _PrecioExtra;
        public int PrecioExtra
        {
            get { return _PrecioExtra; }
            set { _PrecioExtra = value; }
        }
    }
}

File: Scala/src/main/scala/recital/CategoriaBanda.scala
package recital

class CategoriaBanda () {
	var tipo: String = _
	var precioExtra: Double = _
	
	def this(tipo: String, precioExtra: Double) = {
	  this()
	  this.tipo = tipo
	  this.precioExtra = precioExtra
	}
}